webpack.dev.js

```diff
@@ -21,4 +23,21 @@ module.exports = merge(common, {
             }
         ]
     },
+    plugins: [
+        new HtmlWebpackPlugin({
+            filename: 'index.html',
+            template: 'index.html'
+        }),
+        new ConcatPlugin({
+            name: 'scripts/apploader.js',
+            filesToConcat: ['./scripts/apploader.js']
+        })
+    ],
+    devServer: {
```

Comment:
Any chance you could update `readme.md` with some information about building and running the client with your new changes?

Reply:
Done. I'm not sure regarding the note `yarn OR npm` as i think it should be either or, and since `yarn.lock`was committed to the repo i assumed yarn is the way to go.